packages/tools/viewer-alpha/src/viewerElement.ts

```diff
@@ -1,102 +1,455 @@
-import { Logger } from "core/Misc/logger";
-import type { Nullable } from "core/types";
+import type { PropertyValues } from "lit";
 import type { Viewer } from "./viewer";
+
+import { LitElement, css, html } from "lit";
+import { customElement, property, query, state } from "lit/decorators.js";
+
+import { Logger } from "core/Misc/logger";
 import { createViewerForCanvas } from "./viewerFactory";
 
-// TODO: Use https://lit.dev/ to simplify this code and ease maintenance.
+// Icon SVG is pulled from https://fluentuipr.z22.web.core.windows.net/heads/master/public-docsite-v9/storybook/iframe.html?id=icons-catalog--page&viewMode=story

```

Comment:
can we ?

Reply:
I think so. From what I can tell, they are MIT license. https://www.npmjs.com/package/@fluentui/react-icons